hydpy/models/dam_pump.py

```diff
@@ -10,6 +10,8 @@
 long as the water level at a remote location does not exceed another defined threshold
 (|RemoteWaterLevelMaximumThreshold|).  The latter restriction helps to prevent further
 increasing high flow conditions in downstream areas.
+If the MaxForcedDischarge is negative, the pumping process is carried out in reverse
+direction (e.g. for modeling irrigation processes).
```

Comment:
modelling (As far as I am aware, HydPy's documentation is generally in British English.)